crates/ide/src/display/navigation_target.rs

```diff
@@ -269,22 +268,21 @@ impl ToNavFromAst for hir::Trait {
     const KIND: SymbolKind = SymbolKind::Trait;
 }
 
-impl<D> ToNav for D
+impl<D> TryToNav for D
```

Comment:
(not this PR): might make sense to remove plain ToNav alltogether

Reply:
Yeah, that looks to be the case. Looking at the other implementations of `ToNav` whilst they won't `panic`, the `NavigationTarget`'s `name` will be the empty string.